src/components/Slider/Slider.stories.mdx

```diff
@@ -75,10 +75,10 @@ import DSProvider from "../../theme/provider";
 
 # Slider
 
-| Component Version | DS Version |
-| ----------------- | ---------- |
-| Added             | `0.25.4`   |
-| Latest            | `1.4.1`    |
+| Component Version | DS Version   |
+| ----------------- | ------------ |
+| Added             | `0.25.4`     |
+| Latest            | `Prerelease` |
```

Comment:
I think the next version will be 1.4.2 but Marty should verify.

Reply:
Yes, we can release this next week as `1.4.2`.

Reply:
@jackiequach Actually, don't change this before merging.  I like the idea of using a generic value when udpates are made.  We can update to the proper version number as part of the release prep process.